db/migrate/20170523220843_add_hawkular_keys_to_miq_alerts.rb

```diff
@@ -0,0 +1,27 @@
+class AddHawkularKeysToMiqAlerts < ActiveRecord::Migration[5.0]
+  class MiqAlert < ActiveRecord::Base
+    serialize :hawkular_keys
```

Comment:
Yes, but for non hawkular ones it will always be nil... Unless, someday, the other providers start using Hawkular for alerting, then they could use the column.